tests/unit/test_remote_config_util.py

```diff
@@ -31,31 +32,65 @@ def test_lookup_port_failure(self, run_mock):
             lookup_port, 'any_host'
         )
 
-    @patch('prestoadmin.util.service_util.run')
+    @patch('prestoadmin.util.remote_config_util.run')
     def test_lookup_port_not_integer_failure(self, run_mock):
         run_mock.return_value = _AttributeString('http-server.http.port=hello')
         run_mock.return_value.failed = False
+        run_mock.return_value.return_code = 0
+
         self.assertRaisesRegexp(
             ConfigurationError,
-            'Unable to coerce http-server.http.port \'hello\' to an int. '
-            'Failed to connect to any_host.',
+            'Invalid value hello: port must be a number between 1 and 65535.'
```

Comment:
I'd propose that this should match the out of range case and read 'Invalid port number', but if it's a pain to change, it's not worth holding this up over.